mlflow/ml-package-versions.yml

```diff
@@ -3,6 +3,7 @@ sklearn:
     pip_release: "scikit-learn"
     install_dev: |
       # This installs scikit-learn from the default branch
+      echo "dummy change"
```

Comment:
I'll revert this change once I confirm the dev version of sklearn can be installed now.

Reply:
Confirmed dev sklearn can be built & installed with Python 3.7 in [this CI run](https://github.com/mlflow/mlflow/runs/2630145859)

Reply:
Reverted the change in https://github.com/mlflow/mlflow/pull/4380/commits/d1dffc6971de4f7ee8d798798a7f140db1a4aa6e